test/confit-test.js

```diff
@@ -26,6 +26,10 @@ test('confit', function (t) {
 
 
     t.test('get', function (t) {
+        //setting process.env.env to development, should not change 'env:env'.
+        //This should be ignored and 'env:env' should solely depend on process.env.NODE_ENV
+        process.env.env = 'development';
```

Comment:
I assumed that, the existing tests would take care of testing the value of `config.get('env:env')`. Are you suggesting to add more tests to the existing unit tests?

Let me set the `process.env.env` to something other than `development` - i am guessing thats what you are suggesting.